src/parcsr_ls/HYPRE_parcsr_amg.c

```diff
@@ -6,6 +6,7 @@
  ******************************************************************************/
 
 #include "_hypre_parcsr_ls.h"
+#include "protos.h"
```

Comment:
@oseikuffuor1 I don't think we want to do this.  The current model is to introduce `headers` scripts that concatenate files like this into `_hypre_parcsr_ls.h`.  If there isn't already a script, you can manually do it for now.  I am open to changing our header file conventions at some point, but we'll need to discuss as a team first.

Reply:
There is `headers` in this folder. Should run it and not include "protos.h"